### ISSUE TITLE
XL cartridge and jalousie not fit

### ISSUE THREAD

**Issue by Jessi810** [nevermore3d/Nevermore_Micro#55]
The jalousie is 0.2mm thicker than the cartridge's jalousie slider (not sure what to name it, image below for reference).

![2023-11-05 17 11 41](https://github.com/nevermore3d/Nevermore_Micro/assets/16742304/a59850d6-79d2-4797-9bf0-903f1bfc2890)

**Comment by 0ndsk4** [comment]
Yeah, true. Tolerance seems too little. I might have too much squish prototyping :)

Try the two thin/stiff filament jalousie variations if you have a hard time fitting it, They are 0.2mm thinner. Will likely update the cartridge file with a bit more wiggle room. Thanks.
